enterprise/backend/src/metabase_enterprise/serialization/api.clj

```diff
@@ -160,8 +187,22 @@
   [:as {raw-params :params}]
   (api/check-superuser)
   (try
-    (let [{:keys [log-file callback]} (unpack&import (get-in raw-params ["file" :tempfile])
-                                                     (get-in raw-params ["file" :size]))]
+    (let [start              (System/currentTimeMillis)
+          {:keys [log-file
+                  error-message
+                  report
+                  callback]} (unpack&import (get-in raw-params ["file" :tempfile])
+                                            (get-in raw-params ["file" :size]))
+          imported           (set (map (comp :model last) (:seen report)))]
+      (snowplow/track-event! ::snowplow/serialization-import api/*current-user-id*
+                             {:source        "api"
+                              :duration      (long (/ (- (System/currentTimeMillis) start) 1000))
```

Comment:
I find it helpful to include the units in fields like this, e.g. `duration-seconds`

Reply:
done